app/controllers/staffs_controller.rb

```diff
@@ -0,0 +1,39 @@
+class StaffsController < ApplicationController
+
+  before_action :authenticate_customer!
+  
+  def index
+    @staffs = Staff.all(_account_id: current_customer.current_account_id).fetch
+  end
+
+  def new
+    @staff = Staff.new
+  end
+  
+  def create
+    @staff = Staff.new(staff_params.to_h.merge(account_id: current_customer.current_account_id))
+    if @staff.save
+      redirect_to staffs_path
+    else
+      respond_to do |format|
+        format.js { render partial: "staffs/form", locals: { staff: @staff }, replace: ".staff-form-container" }
+      end
+    end
+  end
+  
+  def destroy
+    @staff = Staff.new(id: params[:id], account_id: current_customer.current_account_id)
```

Comment:
@deepali1-aconnexion does it possible to use methods like `where(id: params[:id], account_id: current_customer.current_account_id).first`?